practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -127,12 +134,12 @@ const Analytics = () => {
               height='200px'
             />
           </div>
-        ) : customers ? (
+        ) : customers.length > 0 ? (
           <div className='customer__table'>
             {/* Start sort title */}
             <div className='customer__sort'>
               {SORT_TITLES.map((SORT_TITLE) => (
-                <div className='sort__item col-3' key={uuidv4()}>
+                <div className='sort__item col-3' key={SORT_TITLE.title}>
```

Comment:
are you sure the title is not duplicate?

Reply:
I added more Id to the data of sort title in local to make sure it's not duplicate
![image](https://github.com/datdragon777/React-Training/assets/74806826/fd535e4f-612a-4dd3-abcc-79cfa4564f17)
![image](https://github.com/datdragon777/React-Training/assets/74806826/83288720-900b-4462-94e5-027dd5f1192c)